contracts/Comptroller/Comptroller.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.5.16;
+pragma solidity 0.8.13;
```

Comment:
Do we still need Comptroller.sol? My understanding was that it would be removed in favor of the Diamond Comptroller

Reply:
Yeah, we will move that contract to tests contracts as it will be used in tests(as reference).